cluster/addons/fluentd-elasticsearch/es-image/elasticsearch_logging_discovery.go

```diff
@@ -103,7 +102,7 @@ func main() {
 	namespace := metav1.NamespaceSystem
 	envNamespace := os.Getenv("NAMESPACE")
 	if envNamespace != "" {
-		if _, err := client.CoreV1().Namespaces().Get(context.TODO(), envNamespace, metav1.GetOptions{}); err != nil {
```

Comment:
I don't think these contexts can be removed.

Reply:
its confusing though, I'm unable to build this with the contexts present.  If I remove them, my IDE yells at me but both image build and image execution are successful (ie:  I get a functioning es cluster).

this is the source of the CI errors as well.

do you have any suggestions as to how I can dig into why I'm unable to build with the contexts present?  Is there a client go pinning issue I need to run down?

Reply:
I had to pin to a much later version of kubernetes (this is old code) and then do the module replace dance.  should pass the linting/verify CI checks now.